kola-denylist.yaml

```diff
@@ -117,3 +117,11 @@
   osversion:
     - centos-10
     - rhel-10.1
+
+# We are currently adding a dead symlink to allow MCO to install a pacemaker
+# resource agent in Two Node OpenShift with Fencing (TNF). This will be removed
+# once the underlying agent is shipped as part of the `two-node-ha` extension
+# and the TNF controller in etcd is updated to initialize pacemaker clusters
+# via the version shipped in the extension.
+- pattern: ext.config.shared.files.validate-symlinks
+  tracker: https://issues.redhat.com/browse/OCPEDGE-1706
```

Comment:
Hmm, denylisting that test for an unknown amount of time doesn't feel great.

I think... we can get this in as is for now, but can you also open a PR to add it to https://github.com/coreos/fedora-coreos-config/blob/350e4d59c2ca8dd84419982eb51910961b095f86/tests/kola/files/validate-symlinks#L39 and then we can drop this denylist entry in the next fedora-coreos-config submodule bump.

Reply:
Filed https://issues.redhat.com/browse/OCPEDGE-1752 to track updating the kola test. I can follow up with a fix for that and this repo once that's checked-in.